apps/api/src/services/data-provider/ghostfolio-scraper-api/ghostfolio-scraper-api.service.ts

```diff
@@ -75,12 +77,9 @@ export class GhostfolioScraperApiService implements DataProviderInterface {
       const html = await get();
       const $ = cheerio.load(html);
 
-      const string = $(scraperConfig?.selector)
-        .text()
-        .replace('CHF', '')
```

Comment:
Is this still needed with the regexp extraction?

Reply:
It's red -> deleted code 😄